src/Native/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 2.8.12)
```

Comment:
I think it might make sense to split the C bindings for libgpiod into seperate PR as they look good to me and we should be able to merge it as is. The C# part I still there is some work w.r.t. tear down and multi-threading (although you might want to split the events part from the rest to make it easier to merge and review)

Reply:
Actually it was not so big PR initially, more changes added according to code review comment ...